n26/designsystem/templates/designsystem/demos/deferred/10-fetched-when-first-seen.html

```
@@ -0,0 +1,20 @@
+            @click="expanded = !expanded"
+            :aria-expanded="expanded"
+            class="rounded-control px-2 py-1 text-sm text-accent-text hover:underline focus-ring">
+        <span x-text="expanded ? 'Hide the card' : 'Show the card'">Show the card</span>
+    </button>
+    {% comment %}
+    The x-if is the whole trick: the deferred fetches on its own init,
+    and the template holds init back until the reader asks.
+    {% endcomment %}
+    <template x-if="expanded">
+        <div class="mt-2 max-w-md">
+            <c-n26.deferred url="{% url 'designsystem:sample_fragment' %}" />
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)